--- gnutls-3.3.11/lib/accelerated/cryptodev.c	2015-01-02 19:52:27.573794237 +0800
+++ gnutls-3.3.11/lib/accelerated/cryptodev.c_new	2015-01-02 19:51:57.369793128 +0800
@@ -120,7 +120,7 @@
 	ctx->cryp.src = (void *) src;
 	ctx->cryp.dst = dst;
 	ctx->cryp.op = COP_ENCRYPT;
-	ctx->cryp.flags = COP_FLAG_WRITE_IV;
+	ctx->cryp.flags = (1 << 2); //COP_FLAG_WRITE_IV;
 
 	if (ioctl(ctx->cfd, CIOCCRYPT, &ctx->cryp)) {
 		gnutls_assert();
@@ -140,7 +140,7 @@
 	ctx->cryp.src = (void *) src;
 	ctx->cryp.dst = dst;
 	ctx->cryp.op = COP_DECRYPT;
-	ctx->cryp.flags = COP_FLAG_WRITE_IV;
+	ctx->cryp.flags = (1 << 2); //COP_FLAG_WRITE_IV;
 
 	if (ioctl(ctx->cfd, CIOCCRYPT, &ctx->cryp)) {
 		gnutls_assert();
